findspark.py

```diff
@@ -35,7 +37,74 @@ def find():
     return spark_home
 
 
-def init(spark_home=None):
+def change_rc(spark_home, spark_python, py4j):
+    """Persists changes to enviornment by changing shell config.
+
+    Adds lines to .bashrc to set enviornment variables including
+    the adding of dependencies to the system path. Currently only 
+    works for Bash and (t)csh.
+
+    Parameters
+    ----------
+    spark_home : str
+        Path to Spark installation.
+    spark_python : str
+        Path to python subdirectory of Spark installation.
+    py4j : str
+        Path to py4j library.
+    """
+
+    bashrc_location = os.path.expanduser("~/.bashrc")
+
+    with open(bashrc_location, 'a') as bashrc:
```

Comment:
Let's verify that these shell files already exist before opening them. We don't need to make a cshrc for people who have never used or configured csh (most people). I'd almost think we could ignore csh, since that's what roughly everyone does these days. Do you have a particular reason for configuring csh in addition to bash here?